Program.cs

```diff
@@ -2,16 +2,38 @@
 {
     internal class Program
     {
+        static string Currencyprint() {
+            Console.WriteLine("" +
```

Comment:
1. If your program only accepts those currencies then you should use a different enum for that. You can make use of the same enum but taking a subset of it. [Subset C# Enums](https://codejack.com/2020/06/subset-csharp-enums/)
2. You should print those values dynamically
3. it would be better to have a method that reads the currency type (including the prompt logic) instead of calling a method that returns a string and then decoding it, that way if you had to change the implementation of "Getting the CurrencyType" you wouldn't care about its usages.